src/main/java/com/coveo/configuration/parameterstore/strategy/DefaultParameterStorePropertySourceEnvironmentPostProcessStrategy.java

```diff
@@ -0,0 +1,71 @@
+package com.coveo.configuration.parameterstore.strategy;
+
+import static com.coveo.configuration.parameterstore.ParameterStorePropertySourceConfigurationProperty.SSM_CLIENT_ENDPOINT_CONFIG_ENDPOINT;
+import static com.coveo.configuration.parameterstore.ParameterStorePropertySourceConfigurationProperty.SSM_CLIENT_ENDPOINT_CONFIG_SIGNING_REGION;
+
+import org.springframework.core.env.ConfigurableEnvironment;
+
+import com.amazonaws.client.builder.AwsClientBuilder.EndpointConfiguration;
+import com.amazonaws.regions.AwsRegionProviderChain;
+import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagement;
+import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagementClientBuilder;
+
+import com.coveo.configuration.parameterstore.ParameterStorePropertySource;
+import com.coveo.configuration.parameterstore.ParameterStorePropertySourceConfigurationProperty;
+import com.coveo.configuration.parameterstore.ParameterStoreSource;
+
+public class DefaultParameterStorePropertySourceEnvironmentPostProcessStrategy
```

Comment:
I would rename all the classes in this package since they are called in a post processor but it does not really matter here. What they really do is configure the property source. I'd replace the words `EnvironmentPostProcess` for `Configuration`.

Reply:
For example here it would be `DefaultParameterStorePropertySourceConfigurationStrategy `